adb_graphics/specs.py

```diff
@@ -173,7 +173,7 @@ def lcl_colors(self) -> np.ndarray:
         ''' Default color map for Lifted Condensation Level '''
 
         ctable = ctables.colortables.get_colortable(self.vspec.get('cmap')) \
-                    (range(60, 180, 6)) # rainbow
+                    (range(50, 180, 7)) # rainbow
```

Comment:
was this change something intentional or just a hiccup from our various color table tests?  just curious.

Reply:
I was attempting to bump it into darker blues just a tad to get the white contour at 1000 mb a tad more noticeable. Is that okay?

Reply:
Yeah, looks good, I just thought I might have done something odd in one of my PRs that you were trying to correct.  :-)